Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM openjdk:11-jdk
```

Comment:
You could also use existing Dockerfiles: https://github.com/jenkinsci/jenkinsfile-runner/tree/master/packaging/docker

Reply:
Feel free to create an upstream PR if needed

Reply:
I think I need to explain this thing in detail. @AbhyudayaSharma 
1. I have read the Dockerfiles in jenkinsfile-runner but it cannot meet the needs of the current stage in this project. These Dockerfiles will compile the jenkinsfile-runner from scratch. **So it means if I want to upgrade the core and plugins, I need to fix every versions in the parent pom.** This will **overwrite every old containers** of jenkinsfile-runner.
2. It's really hard for me to figure out the version dependency between plugins and core and even among plugins.
3. Oleg is working on jenkinsfile-runner java11 version upgrading. Currently, I still don't know his opinions about whether I can upgrade the jenkins core and plugins in the jenkinsfile-runner official image.
4. At present, the users almost cannot install new plugins in the container because it would cause version conflicts exception. And I cannot go on to add more new demos because I cannot install some plugins.

Reply:
@oleg-nenashev would it be okay for Yiming to create a PR upgrading JFR's dependencies?

Reply:
@Cr1t-GYM At this point I would suggest you go ahead and open a PR at https://github.com/jenkinsci/jenkinsfile-runner and ping Oleg for a review and go from there.